app/Http/Controllers/Web/ApplicationController.php

```diff
@@ -45,13 +46,13 @@ public function platformRedirect(Application $application, Agent $agent, Platfor
      * @param Request $request
      * @param Application $application
      * @param Platform $platform
-     *
+     * @param Build|null $build
      * @param Cloud $storage
      * @return RedirectResponse
      */
-    public function install(Request $request, Application $application, Platform $platform, Cloud $storage): RedirectResponse
+    public function install(Request $request, Application $application, Platform $platform, ?Build $build, Cloud $storage): RedirectResponse
```

Comment:
```suggestion
    public function install(Request $request, Application $application, Platform $platform, ?Build $build = null, Cloud $storage): RedirectResponse
```

Because here you are missing the default

Reply:
> Actually no, it's not null if it doesn't fetch it, it's still a model instance but it's empty, I tried. Doesn't make sense really

Reply:
Nope because we're not getting that from the request. It's injected as a service (from what I saw)

Reply:
I thought PHP gives warning if parameters which goes after a one with the default value don't have default values.

Reply:
Exactly

Reply:
Done in https://github.com/everli/uppy/pull/43/commits/ff82d3f71a44a10e08b70d171874fee8a27218dd